### PR TITLE
Oop blackjack game

### PR DESCRIPTION
**Pull request by dianedowling**
For code review! Sections of this code will then be used as "real code" on the platform.

### CODE REVIEW

File: object-oriented-programming/blackjack-game/python/blackjack.py [hunk 1]
```diff
@@ -0,0 +1,195 @@
...
+# Usage licensed under the Open Government Licence v3.0
+# BETA VERSION
+
+from player_class import Player
+from deck_class import Deck
+import time
+
+
+class BlackJack:
+    """A simple version of the game of BlackJack"""
```
Comment (alexparry) [inline object-oriented-programming/blackjack-game/python/blackjack.py:11]: Remove "simple" ?

**Review by alexparry — COMMENTED** [review]
Done! The main thing in general is about adding comments. I think that most methods should have a docstring as well as general comments within the methods

File: object-oriented-programming/blackjack-game/python/blackjack.py (hunk 1 above)
Comment (lawsie) [reply object-oriented-programming/blackjack-game/python/blackjack.py:11]: Simplified?

File: object-oriented-programming/blackjack-game/python/hand_class.py [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+        self.__cards = []
+        self.__value = 0
+        self.__aces = 0
+
+    def add_card(self, card):
+        self.__cards.append(card)
+        if card.get_rank() == "Ace":
+            self.__aces += 1
+        self.__value += card.get_value()
+        if self.__value > 21: # adjust for aces
```
Comment (lawsie) [inline object-oriented-programming/blackjack-game/python/hand_class.py:17]: I think this needs some more explanation as to what you are doing with regard to the value of the aces

Comment (lawsie) [reply object-oriented-programming/blackjack-game/python/hand_class.py:17]: Also, what happens in the eventuality that you originally wanted an ace to be 11 and then after twisting you actually want it to be 1?

**Comment by lawsie** [comment]
I agree with Alex that there could generally be more comments to explain stuff, and/or docstrings as necessary

File: object-oriented-programming/blackjack-game/python/hand_class.py (hunk 2 above)
Comment (dianedowling) [reply object-oriented-programming/blackjack-game/python/hand_class.py:19]: That's ok as it re-evaluates each time nd makes best choice.

**Pull request merged**